ddl/placement/bundle.go

```diff
@@ -288,27 +288,13 @@ func (b *Bundle) String() string {
 
 // Tidy will post optimize Rules, trying to generate rules that suits PD.
 func (b *Bundle) Tidy() error {
-	extraCnt := map[PeerRoleType]int{}
-	newRules := b.Rules[:0]
-
-	// One Bundle is from one PlacementSettings, rule share same location labels, so we can use the first rule's location labels.
-	var locationLabels []string
+	tempRules := b.Rules[:0]
+	id := 0
 	for _, rule := range b.Rules {
```

Comment:
Could you add some extra checks for rules in Tidy? I think `Tidy` can only be called when some fields are empty (`StartKeyHex/EndKeyHex` .etc).

Reply:
As the comments saied, it's optimized rules, try to simplify the rules.